manifests/plugin/gutterball/config.pp

```diff
@@ -0,0 +1,15 @@
+# gutterball config
+class katello::plugin::gutterball::config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/etc/foreman/plugins',
+    $url = "https://${fqdn}:8443/gutterball",
+  ){
+
+  file { "${foreman_plugins_dir}/foreman_gutterball.yaml":
```

Comment:
I agree with @ehelms. the name of the plugin is `foreman_gutterball` so the plugin's config file should have the same name.